apps/web/modules/survey/survey-templates/lib/survey.ts

```diff
@@ -0,0 +1,109 @@
+import { Prisma, Survey } from "@prisma/client";
+import { prisma } from "@formbricks/database";
+import { segmentCache } from "@formbricks/lib/cache/segment";
+import { capturePosthogEnvironmentEvent } from "@formbricks/lib/posthogServer";
+import { surveyCache } from "@formbricks/lib/survey/cache";
+import { getInsightsEnabled } from "@formbricks/lib/survey/utils";
```

Comment:
these two util functions are still being imported from the surveys lib package, pls look into if these can be moved locally as well 🙏